packages/client/src/generation/TSClient/PrismaClient.ts

```diff
@@ -95,8 +95,49 @@ function clientExtensionsQueryDefinition(this: PrismaClientClass) {
     }, '')}
   }
 }`
+}
+
+function clientTypeMapOthersDefinition(this: PrismaClientClass) {
+  const otherOperationsNames = this.dmmf.getOtherOperationNames().flatMap((n) => {
+    if (n === 'executeRaw' || n === 'queryRaw') {
+      return [`$${n}Unsafe`, `$${n}`]
+    }
+
+    return `$${n}`
+  })
+
+  const argsResultMap = {
+    $executeRaw: { args: '[query: TemplateStringsArray | Prisma.Sql, ...values: any[]]', result: 'any' },
+    $queryRaw: { args: '[query: TemplateStringsArray | Prisma.Sql, ...values: any[]]', result: 'any' },
+    $executeRawUnsafe: { args: '[query: string, ...values: any[]]', result: 'any' },
+    $queryRawUnsafe: { args: '[query: string, ...values: any[]]', result: 'any' },
+    $runCommandRaw: { args: 'Prisma.InputJsonObject', result: 'Prisma.JsonObject' },
```

Comment:
I wonder if `Prisma.InputJsonObject` isn't too narrow here. I believe things like dates and buffers should be allowed there but aren't allowed by `InputJsonObject`?

Reply:
the type matches the API without extensions, so out of scope
https://github.com/prisma/prisma/blob/1e0e86795c1d73e5d109c44addb289a411d72e15/packages/client/src/generation/TSClient/PrismaClient.ts/#L439